specs/code-insiders/compare_two_files_in_vs_code_insiders.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Compare two files in VS Code (Insiders)
+command: "code-insiders -d {{file1}} {{file2}}"
+tags: ["code-insiders"]
+description: Compare two files using code.
+arguments:
+  - name: file1
+    description: The first file that you want to compare.
+    default_value: ~
```

Comment:
I think we should remove the default value arguments for this workflow since there doesn't seem to be a good default. The possibilities for a default argument are rather broad and user / workspace dependent

Reply:
done!